doc/source/examples/ale_regression_california.ipynb

```diff
@@ -0,0 +1,875 @@
+{
```

Comment:
<p>I will probably mention explicitly that we are talking about average prediction of the data points which have their <code>MedInc</code> value within that bin.</p><p> </p><p>interpeted -&gt; interpreted</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Added a Note box explaining this.</p>